src/extensions/IPosisSpawnExtension.d.ts

```diff
@@ -17,8 +17,8 @@ interface IPosisSpawnExtension {
     spawnCreep(rooms: string[], body: string[][], memory?: any, opts?: { priority?: number }): string;
```

Comment:
remove `memory` parameter, if a process really needs to set it, they can do it on the returned creep or through Memory.creeps (Although both ways are heavily discouraged)